src/lib/layer-helpers.js

```diff
@@ -25,3 +28,10 @@ export const getLayersTags = compose(
   flatten,
   map(props([ 'tags' ])),
 )
+
+export const getLayersById = (layers, ids) => 
+  compose(
+    filter(idIncludedIn(ids)),
+    flatten,
+    pickLayersRecursive,
+  )(layers)
```

Comment:
missing newline

Reply:
don't we have a linting tool to fix that?

Reply:
We use ESLint, that can/should take care of it. Maybe add the `'eol-last'` rule just to be sure?